### PR TITLE
Add Laravel Deployment Tutorial

### PR DESCRIPTION
**Pull request by engahmedgad**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by:
Ahmed Gad <eng.ahmedmgad@gmail.com>

### CODE REVIEW

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+* Choose the resources you need from type
+* Add your local machine SSH Key 
+    * You can read [This](https://help.github.com/en/enterprise/2.16/user/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) article to know how to generate SSH Key
+* Write your server hostname in `name` Input
+* Click `Create & Buy Now`
+ 
+
+## Step 2 - login into you server
+* Go to servers list in console cloud and copy your server  `IP Address` 
+* Open your  terminal and write ` ssh root@<IP Address>`
```
Comment (n-se) [inline tutorials/deploy-laravel-on-hetzner-cloud/en.md]: You login as root but always use `sudo` below.

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## Step 5 - Install PHP 
+* `sudo apt-get install software-properties-common`
+* `sudo add-apt-repository ppa:ondrej/php` to add php7.1 repository 
+* `sudo apt-get update` to update your package manager
+* install php and required extensions 
+```
+sudo apt-get install php7.1 php7.1-xml php7.1-mbstring php7.1-mysql php7.1-json php7.1-curl php7.1-cli php7.1-common php7.1-mcrypt php7.1-gd libapache2-mod-php7.1 php7.1-zip php7.1-dom
+``` 
+## Step 6 - Install Composer
+* `sudo apt-get install composer`
```
Comment (n-se) [inline tutorials/deploy-laravel-on-hetzner-cloud/en.md]: You first install PHP from ppa to get a newer version but than install an old composer version? Maybe you can add some lines on how to [get the latest version of composer](https://getcomposer.org/doc/00-intro.md#globally)

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md (hunk 1 above)
Comment (engahmedgad) [reply tutorials/deploy-laravel-on-hetzner-cloud/en.md]: I updated this in new pull request #66

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md (hunk 2 above)
Comment (engahmedgad) [reply tutorials/deploy-laravel-on-hetzner-cloud/en.md]: I updated this in new pull request #66

**Comment by n-se** [comment]
Looks good so far. Can you please remove the `.idea` folder, it is left from your editor.
Also please re-read the tutorial again, there are some typos and you should use correct uper- and lowercase. The tags at the top should also start with an uppercase.

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md (hunk 1 above)
Comment (n-se) [reply tutorials/deploy-laravel-on-hetzner-cloud/en.md]: You added `$` in front of the commands. The root user has a `#` at the beginning of the bash line. `$` indicates a user.

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* Click `Add Server` and select `Ubuntu 18.04` from Image
+* Choose the resources you need from Type
+* Add your local machine SSH Key 
+    * You can read [this](https://help.github.com/en/enterprise/2.16/user/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) article to know how to generate SSH key
+* Write your server hostname in `name` input
+* Click `Create & Buy Now`
+ 
+
+## Step 2 - Login into you server
+* Go to servers list in console cloud and copy your server  `IP Address`
```
Comment (n-se) [inline tutorials/deploy-laravel-on-hetzner-cloud/en.md]:
```suggestion
* Go to servers list in console cloud and copy your servers IP address (e.g. 10.0.0.1)
```

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+* Choose the resources you need from Type
+* Add your local machine SSH Key 
+    * You can read [this](https://help.github.com/en/enterprise/2.16/user/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) article to know how to generate SSH key
+* Write your server hostname in `name` input
+* Click `Create & Buy Now`
+ 
+
+## Step 2 - Login into you server
+* Go to servers list in console cloud and copy your server  `IP Address` 
+* Open your terminal and write `# ssh root@<IP Address>`
```
Comment (n-se) [inline tutorials/deploy-laravel-on-hetzner-cloud/en.md]:
```suggestion
* Open your terminal and write `# ssh root@10.0.0.1`
```

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md (hunk 1 above)
Comment (engahmedgad) [reply tutorials/deploy-laravel-on-hetzner-cloud/en.md]: Fixed

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+* Install Composer globally
+    * `# php composer-setup.php --install-dir=/usr/local/bin --filename=composer`
+* Now let's verify that is Composer is installed successfully type in your terminal `# composer` if you see something like `Composer version 1.8.4 2019-02-11 10:52:10
+` then everything is ok
+
+## Step 7 - Install Laravel project
+* `# cd /var/www/html`
+* `# composer create-project --prefer-dist laravel/laravel blog "5.6.*"`
+* Now let's move the files from `blog/` directory to Apache's root directory `# cd blog/ && mv * ../ && mv .env ../`
+* Go to `http://<IP Address>/public` and you will see your app Live now
```
Comment (n-se) [inline tutorials/deploy-laravel-on-hetzner-cloud/en.md]:
```suggestion
* Go to `http://10.0.0.1/public` and you will see your app now
```

Comment (n-se) [reply tutorials/deploy-laravel-on-hetzner-cloud/en.md]: Are you sure that `/public` is correct here?

File: tutorials/deploy-laravel-on-hetzner-cloud/en.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+---
+path: "tutorials/deploy-laravel-on-hetzner-cloud"
+date: "2019-03-08"
+title: "Deploy Laravel App on Hetzner Cloud"
+short_description: "In this article we well learn how to install LAMP stack on Hetzner cloud server and deploy Laravel app on it"
```
Comment (n-se) [inline tutorials/deploy-laravel-on-hetzner-cloud/en.md]:
```suggestion
short_description: "In this article we will learn how to install a LAMP stack on a Hetzner cloud server and deploy a Laravel app on it"
```

**Comment by engahmedgad** [comment]
Is everything ok now ?

**Comment by n-se** [comment]
Looks good to me. A Community Manager will review your tutorial probably soon.

A tip for the future, you can use `Add suggestion to batch` in order to commit all review suggestions in one commit instead of committing each suggestion in a single one.

**Comment by engahmedgad** [comment]
thanks for the tip

**Comment by Billito00** [comment]
Referring to our Contribution Guidelines you have to insert the license agreement before we can publish the tutorial. Please read the guidelines carefully again.

**Comment by engahmedgad** [comment]
I'm sorry but I couldn't understand where to insert license agreement
and I read the contributions guidelines again now but I couldn't see the part of the license
so can you gide me through that?

**Comment by Billito00** [comment]
No worries. Of course we'll help you.
Please add the following lines at the end of your tutorial. 

```
##### License: MIT

<!---

Contributors's Certificate of Origin

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have
    the right to submit it under the license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my
    knowledge, is covered under an appropriate license and I have the
    right under that license to submit that work with modifications,
    whether created in whole or in part by me, under the same license
    (unless I am permitted to submit under a different license), as
    indicated in the file; or

(c) The contribution was provided directly to me by some other person
    who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are
    public and that a record of the contribution (including all personal
    information I submit with it, including my sign-off) is maintained
    indefinitely and may be redistributed consistent with this project
    or the license(s) involved.

Signed-off-by: [submitter's name and email address here]

-->
```

Make sure to sign the licence with your name and email address at the end.

**Comment by engahmedgad** [comment]
Thanks a lot for your help
is everything's okay now?!

**Pull request merged**